### PR TITLE
add ut about schedule policy

### PR DESCRIPTION
**Pull request by yt-huang**
What type of PR is this?

/kind test

ref #569

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

**Review by wawa0210 — APPROVED** [review]

File: pkg/scheduler/policy/node_policy_test.go [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+func TestLess(t *testing.T) {
+	tests := []struct {
+		name          string
+		nodeScoreList NodeScoreList
+		i             int
+		j             int
+		expected      bool
+	}{
+		{
+			name: "Spread 策略，i 分数更高",
```
Comment (wawa0210) [inline pkg/scheduler/policy/node_policy_test.go]: english first

**Comment by wawa0210** [comment]
thanks your contribution

File: pkg/scheduler/policy/node_policy_test.go (hunk 1 above)
Comment (yt-huang) [reply pkg/scheduler/policy/node_policy_test.go]: It's been fixed and re-pushed

**Review by wawa0210 — APPROVED** [review]

**Comment by yt-huang** [comment]
lint has been manually adjusted and executed

**Review by wawa0210 — APPROVED** [review]

**Pull request merged**